gen/pb-go/flyteidl/service/flyteadmin/api/swagger.yaml

```diff
@@ -475,6 +475,25 @@ paths:
           description: "A successful response."
           schema:
             $ref: "#/definitions/adminExecutionCreateResponse"
+  /api/v1/executions/recover:
+    post:
+      tags:
+      - "AdminService"
+      description: "Recovers a previously-run workflow execution that failed due to\
+        \ system failure."
```

Comment:
Does system failure mean a set of error codes . If yes ,  can we  add those in the description?

Reply:
just all errors for now